roboteam_ai/src/skills/SideAttacker.cpp

```diff
@@ -0,0 +1,75 @@
+//
+// Created by thijs on 17-12-18.
+//
+
+#include "SideAttacker.h"
+
+namespace rtt {
+namespace ai {
+
+SideAttacker::SideAttacker(string name, bt::Blackboard::Ptr blackboard)
+        :Skill(name, blackboard) {
+}
+
+/// Init the GoToPos skill
+void SideAttacker::onInitialize() {
+    robot = getRobotFromProperties(properties);
+}
+
+/// Get an update on the skill
+bt::Node::Status SideAttacker::onUpdate() {
+    updateRobot();
+    if (! robot) return Status::Running;
+    Vector2 ball = World::getBall().pos;
+    Vector2 behindBall = Coach::getPositionBehindBall(0.5);
+    Vector2 deltaBall = behindBall - ball;
+    if (! Control::pointInTriangle(robot->pos, ball, ball + (deltaBall).rotate(M_PI*0.17).scale(2.0),
```

Comment:
what does this exactly check? Maybe add some comments at complex lines to help other readers understand the code faster.

Reply:
agreed

Reply:
I will do that later though, currently not really working on sideattacker.